indico/MaKaC/badgeDesignConf.py

```diff
@@ -132,34 +75,44 @@ class BadgeDesignConfiguration:
-                               "Full Name":               (_("Full Name"), Registrant.getFullName),
-                               "Full Name (w/o title)":   (_("Full Name (w/o title)"), RegistrantFullName2),
-                               "Full Name B":             (_("Full Name B"), RegistrantFullName3),
-                               "Full Name B (w/o title)": (_("Full Name B (w/o title)"), RegistrantFullName4),
-                               "Full Name C":             (_("Full Name C"), RegistrantFullName5),
-                               "Full Name C (w/o title)": (_("Full Name C (w/o title)"), RegistrantFullName6),
-                               "First Name":              (_("First Name"), Registrant.getFirstName),
-                               "Surname":                 (_("Surname"), Registrant.getSurName),
-                               "Position":                (_("Position"), Registrant.getPosition),
-                               "Institution":             (_("Institution"), Registrant.getInstitution),
-                               "Country":                 (_("Country"), RegistrantCountry),
-                               "City":                    (_("City"), Registrant.getCity),
-                               "Address":                 (_("Address"), Registrant.getAddress),
-                               "Phone":                   (_("Phone"), Registrant.getPhone),
-                               "Fax":                     (_("Fax"), Registrant.getFax),
-                               "Email":                   (_("Email"), Registrant.getEmail),
-                               "Personal homepage":       (_("Personal homepage"), Registrant.getPersonalHomepage),
-                               "Amount":                  (_("Amount"), Registrant.getTotal),
-                               "Conference Name":         (_("Conference Name"), Conference.getTitle),
-                               "Conference Dates":        (_("Conference Dates"), ConferenceDates),
-                               "Fixed Text":              (_("Fixed Text"), BadgeTemplateItem.getFixedText)
-                            }
-
-            """ Dictionary that maps group names to the item names that fall into that group.
-            The groups are only used for the <select> box in the WConfModifBadgeDesign.tpl file.
-            """
-            self.groups = [( _("Registrant Data"), [ "Title", "Full Name", "Full Name (w/o title)", "Full Name B", "Full Name B (w/o title)", "Full Name C", "Full Name C (w/o title)", "First Name",  "Surname",  "Position",
-                                                     "Institution",  "Country",  "City", "Address", "Phone", "Fax", "Email", "Personal homepage", "Amount"]),
-                      ( _("Conference Data"), [ "Conference Name",  "Conference Dates"]),
-                      ( _("Fixed Elements"), [ "Fixed Text"])]
+        title_lambda = lambda x: x.get_personal_data()['title']
```

Comment:
It's simply not considered clean (since a named lambda is just a more restricted version of a `def`).

Does it look unclean even if you wrap after the `{` of the dict?